subitem.go

```diff
@@ -47,30 +52,51 @@ type SubscriptionItemBillingThresholdsParams struct {
 // SubscriptionItemListParams is the set of parameters that can be used when listing invoice items.
 // For more details see https://stripe.com/docs/api#list_invoiceitems.
 type SubscriptionItemListParams struct {
-	ListParams   `form:"*"`
-	Subscription *string `form:"subscription"`
+	EndingBefore  *string   `form:"ending_before"`
+	Expand        []*string `form:"expand"`
+	Limit         *int64    `form:"limit"`
+	StartingAfter *string   `form:"starting_after"`
+	Subscription  *string   `form:"subscription"`
+}
+type SubscriptionItemUsageRecordSummariesParams struct {
+	Params        `form:"*"`
+	EndingBefore  *string `form:"ending_before"`
+	Limit         *int64  `form:"limit"`
+	StartingAfter *string `form:"starting_after"`
+}
+type SubscriptionItemCreateUsageRecordParams struct {
+	Params           `form:"*"`
+	SubscriptionItem *string `form:"-"` // Included in URL
+	Action           *string `form:"action"`
+	Quantity         *int64  `form:"quantity"`
+	Timestamp        *int64  `form:"timestamp"`
+}
+type SubscriptionItemListUsageRecordSummariesParams struct {
+	ListParams       `form:"*"`
+	SubscriptionItem *string `form:"-"` // Included in URL
 }
 
 // SubscriptionItem is the resource representing a Stripe subscription item.
 // For more details see https://stripe.com/docs/api#subscription_items.
 type SubscriptionItem struct {
 	APIResource
-	BillingThresholds SubscriptionItemBillingThresholds `json:"billing_thresholds"`
-	Created           int64                             `json:"created"`
-	Deleted           bool                              `json:"deleted"`
-	ID                string                            `json:"id"`
-	Metadata          map[string]string                 `json:"metadata"`
-	Plan              *Plan                             `json:"plan"`
-	Price             *Price                            `json:"price"`
-	Quantity          int64                             `json:"quantity"`
-	Subscription      string                            `json:"subscription"`
-	TaxRates          []*TaxRate                        `json:"tax_rates"`
+	BillingThresholds *SubscriptionItemBillingThresholds `json:"billing_thresholds"`
```

Comment:
Damn that one is a breaking change so it will need a todo/override

Reply:
Added the override!